vendor.conf

```diff
@@ -14,18 +14,18 @@ github.com/prometheus/common ebdfc6da46522d58825777cf1f90490a5b1ef1d8
 github.com/prometheus/procfs abf152e5f3e97f2fafac028d2cc06c1feb87ffa5
 
 github.com/docker/distribution 7230e9def796c63a4033211dc5107742d689fc1e
-github.com/docker/docker 0fb0d67008157add34f1e11685e23a691db92644
+github.com/docker/docker e4512d264741e83e954a19f9ef5e3cb06c5856b6
```

Comment:
What version is this coming from?

Reply:
so the problem is the following:
the original commit does not contain an api that libnetwork uses.
This is the commit id that is used inside libnetwork: https://github.com/docker/libnetwork/blob/release/v0.9/Godeps/Godeps.json#L147

But unfortunately the latter does not have anymore the types: types.ContainersPruneConfig, types.ImagesPruneConfig, types.NetworksPruneConfig, types.VolumesPruneConfig
That got nuked by this commit: https://github.com/moby/moby/pull/28535